algorithms/kernel/svm/svm_train_common_avx512_impl.i

```diff
@@ -109,101 +108,95 @@ void SVMTrainTask<float, daal::algorithms::svm::interface2::Parameter, avx512>::
         __m512 objFunc_vec = _mm512_div_ps(b_vec, a_vec);       // b/a = delta.
         objFunc_vec        = _mm512_mul_ps(objFunc_vec, b_vec); // we have objFunc = b * b / a
 
-        // generally, objFunc = -b^2/a.
-        // if (objFunc <= GMin) -> if (-objFunc > -GMin)
-        __mmask16 mask4_ = _mm512_cmp_ps_mask(objFunc_vec, valMinusGMin, _CMP_GE_OS);
-        mask4_           = _mm512_kand(mask2_, mask4_);                           // combine with previous masks
-        valMinusGMin     = _mm512_mask_mov_ps(valMinusGMin, mask4_, objFunc_vec); // -GMin = b^2/a
-        Bj_vec           = _mm512_mask_mov_epi32(Bj_vec, mask4_, Bj_vec_cur);     // result index
+        // generally, objFunc = b^2/a.
+        // if (objFunc > GMax)
+        __mmask16 mask4_ = _mm512_cmp_ps_mask(objFunc_vec, valGMax, _CMP_GE_OS);
+        mask4_           = _mm512_kand(mask2_, mask4_);                       // combine with previous masks
+        valGMax          = _mm512_mask_mov_ps(valGMax, mask4_, objFunc_vec);  // GMax = b^2/a
+        Bj_vec           = _mm512_mask_mov_epi32(Bj_vec, mask4_, Bj_vec_cur); // result index
     }
 
     // reduction:
-    TNArray<float, 16, avx512> minus_GMin_val(16);
-    TNArray<int, 16, avx512> Bj_val_overall(16);
-    TNArray<float, 16, avx512> minusGMin2_overall(16);
+    float * GMaxVal      = reinterpret_cast<float *>(&valGMax);
```

Comment:
minor comment to follow codding style:
`float * const GMaxVal `
similar for bellow